.github/workflows/test.yaml

```diff
@@ -0,0 +1,51 @@
+name: Test
```

Comment:
Will this file be merged with this PR?

Reply:
Yes, that's the idea, but I'm failing to make it work.

Reply:
https://github.com/orgs/community/discussions/25288#discussioncomment-3247293

> To trigger a workflow runs on the workflow_run event, the workflow file should exist on the master or default branch.

Reply:
That answer is from 2020. It makes sense for this to be triggerable from branches other than `main`/`master`. I haven't read about it. I'm okay with merging if you want to try it out.